### ISSUE TITLE
training question

### ISSUE THREAD

**Issue by MissAimee** [hq-jiang/instance-segmentation-with-discriminative-loss-tensorflow#6]
@hq-jiang Thank you for your code, I am confused about your training pipeline. Do you means training step by step? And how to do it ?
